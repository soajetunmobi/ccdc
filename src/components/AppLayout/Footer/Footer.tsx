import React, { FC } from 'react'

export const Footer: FC = () => {
  const currentYear = new Date().getFullYear()

  return (
    <footer className='footer'>
      <span>CCDC</span>
      <br />
      {currentYear}
    </footer>
  )
}
