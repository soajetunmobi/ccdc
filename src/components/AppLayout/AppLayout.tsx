import React, { FC, PropsWithChildren, ReactNode } from 'react'
import { Header } from './Header'
import { Main } from './Main'
import { Footer } from './Footer'

export const AppLayout: FC = ({ children }: PropsWithChildren<ReactNode>) => {
  return (
    <>
      <Header />
      <Main>{children}</Main>
      <Footer />
    </>
  )
}
