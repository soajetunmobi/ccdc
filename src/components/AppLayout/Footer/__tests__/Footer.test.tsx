import React from 'react'
import { Footer } from 'components/AppLayout/Footer'
import { render } from '@testing-library/react'

test('it renders Footer correctly', () => {
  const { container } = render(<Footer />)
  expect(container).toMatchSnapshot()
})
