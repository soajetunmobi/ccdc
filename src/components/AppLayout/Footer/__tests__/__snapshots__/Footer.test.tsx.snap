// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Footer correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <span>
      CCDC
    </span>
    <br />
    2021
  </footer>
</div>
`;
